src/Npgsql/NpgsqlConnection.cs

```diff
@@ -1576,9 +1577,51 @@ public override DataTable GetSchema()
         /// in the Restrictions collection.
         /// </param>
         /// <returns>The collection specified.</returns>
-        public override DataTable GetSchema(string? collectionName, string?[]? restrictions)
+        public override DataTable GetSchema(string collectionName, string[] restrictions)
```

Comment:
The nullability here is `string?[]?` (but we should really just wait for a preview which will flag all this as errors)

Reply:
Changed back to initial situation.